website/source/docs/runtime/interpolation.html.md.erb

```diff
@@ -149,11 +149,11 @@ Below is a table documenting common node properties:
     <td>Version of the client kernel (e.g. <tt>3.19.0-25-generic</tt>, <tt>15.0.0</tt>)</td>
   </tr>
   <tr>
-    <td><tt>${attr.platform.aws.ami-id}</tt></td>
+    <td><tt>${attr.unique.platform.aws.ami-id}</tt></td>
```

Comment:
This PR fixes this: https://github.com/hashicorp/nomad/pull/2999/files#diff-fa62573917c2ea35cf929a8f88b36f20R101